test/cfg/windows.cpp

```diff
@@ -1144,17 +1144,6 @@ void GetShortPathName_validCode(const TCHAR* lpszPath)
     delete[] buffer;
 }
 
-class MyClass : public CObject {
```

Comment:
This is not duplicated so it should be moved to `mfc.cpp`.

Reply:
Side note: we should add a CI compilation step for `windows.cpp`, it totally does not compile at all.

Reply:
> Side note: we should add a CI compilation step for `windows.cpp`, it totally does not compile at all.

I was wondering about that...